app/views/organizations/_table.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="table-responsive">
+  <table class="table table-striped data-table">
+    <thead>
+      <tr>
+        <th class="sort-asc">County</th>
+        <th>Name</th>
+        <th>Mailing Address</th>
+        <th><%= t :organization_phone %></th>
+        <th>Email</th>
+        <th>Programs</th>
+
+        <% if current_user.can_delete_and_restore_organizations? %>
+          <th>Actions</th>
+        <% end %>
+      </tr>
+    </thead>
+    <tbody>
+      <% @organizations.each do |organization| %>
+        <tr data-href="<%= edit_organization_path(organization) %>">
+          <td><%= organization.county %></td>
+          <td><%= organization.name %></td>
+          <td><%= organization.primary_address %></td>
+          <td><%= organization.phone_number %></td>
+          <td><%= organization.email %></td>
+          <td>
+            <span title="<%= organization.programs.map(&:name).sort.to_sentence %>" data-toggle="tooltip">
+              <%= organization.programs.map(&:initialized_name).sort.to_sentence %>
```

Comment:
it's possible these will give different orders

Reply:
good call. Not a blocker to me but we should make initialize name a view helper maybe and just use name array.

Reply:
@tamouse Please feel free to make a ticket for this and make it a quickfix 👍